pages/api/notification.ts

```diff
@@ -0,0 +1,57 @@
+import { Events, parseAndVerifyRequest } from '@nft/webhook'
+import { NextApiRequest, NextApiResponse } from 'next'
+import nodemailer, { SendMailOptions } from 'nodemailer'
+import invariant from 'ts-invariant'
+import AuctionBidCreated from '../../emails/AuctionBidCreated'
+import BidCreated from '../../emails/BidCreated'
+
+const transporter = nodemailer.createTransport({
```

Comment:
Shouldn't we add checks to make sure the envs are properly set?
If they are not set, it try to send using localhost.

```suggestion
invariant(process.env.EMAIL_HOST, 'env EMAIL_HOST is required')
invariant(process.env.EMAIL_PORT, 'env EMAIL_PORT is required')
invariant(process.env.EMAIL_USERNAME, 'env EMAIL_USERNAME is required')
invariant(process.env.EMAIL_PASSWORD, 'env EMAIL_PASSWORD is required')

const transporter = nodemailer.createTransport({
```

Reply:
I think having it optional could make sense but ok to force it. In general I would like to also stop to rely on the global state for these kind of check but right now we have it this way so let's continue like that